spring-boot-testjars-maven/src/test/java/org/springframework/experimental/boot/server/exec/MavenClasspathEntryTests.java

```diff
@@ -103,29 +103,29 @@ void resolveDependencyWhenCustomRepository() {
 		List<RemoteRepository> repositories = new ArrayList<>();
 		repositories.add(
 				new RemoteRepository.Builder("central", "default", "https://repo.maven.apache.org/maven2/").build());
-		repositories.add(new RemoteRepository.Builder("sonatype-snapshot", "default",
-				"https://oss.sonatype.org/content/repositories/snapshots/").build());
```

Comment:
I was unable to reach Sonatype so figured we could point at something a little less "blocky".